modules/heat_transfer/doc/content/source/physics/HeatConductionFE.md

```diff
@@ -0,0 +1,19 @@
+# Heat Conduction Finite Element
```

Comment:
I'd just keep it as "HeatConductionFE" to match the class name

Reply:
Syntax is HeatConduction/FiniteElement so it's more searchable that way?
I think if you are coming from a moose input to the documentation this way is nice
If you are coming from the source code and wanting to look at the doco, HeatConductionFE is easier

Reply:
I guess for Diffusion I did not do it. As you want.

Reply:
Yeah it s the 'source' index so we ought to be consistent with the source

the syntax page can be consistent with the syntax.